baseapp/metrics.go

```diff
@@ -81,7 +81,7 @@ func RegisterDefaultMetrics(registry metrics.Registry) {
 
 // CountRequest is an hlog access handler that records metrics about the
```

Comment:
We should fix these comments to remove the `hlog` references since that is no longer true.